applications/templates/dashboard.html

```diff
@@ -50,12 +50,25 @@
       </div>
     {% elif application.is_attended %}
       <div class="col-md-12">
-        <p class="lead">Welcome to {{ h_name }}. Let us know if you need any help!</p>
-        {% if h_live %}
-          <p>Follow our live page for more information during the event</p>
-          <a href="{{ h_live }}" target="_blank" class="btn btn-block btn-success">Go to live page</a>
-
-        {% endif %}
+        <p class="lead text-center">Welcome to {{ h_name }}. Here are useful resources to have handy throughout the event: </p>
```

Comment:
I will most likely be re-factoring this sense I will have to include this verbiage in the html email templates; however, works for now!